openforcefield/tests/utils.py

```diff
@@ -20,12 +20,59 @@
 import os
 import pprint
 import textwrap
+import importlib
 
+import pytest
 import numpy as np
 from simtk import unit, openmm
 
 from openforcefield.utils import get_data_file_path
+from openforcefield.utils import (
+    OpenEyeToolkitWrapper,
+    RDKitToolkitWrapper,
+    AmberToolsToolkitWrapper,
+)
+
+
+requires_rdkit = pytest.mark.skipif(
+    not RDKitToolkitWrapper.is_available(), reason='Test requires RDKit',
+)
+requires_openeye = pytest.mark.skipif(
+    not OpenEyeToolkitWrapper.is_available(), reason='Test requires OE toolkit',
+)
+requires_openeye_mol2 = pytest.mark.skipif(
+    requires_openeye.args, reason = 'Test requires OE toolkit to read mol2 files'
+)
+
```

Comment:
These could be condensed down a little bit if there is no functional difference between a toolkit being import-able and being available. It looks like there is some complexity in `OpenEyeToolkitWrapper.is_available` so I'm electing to not condense this

Reply:
Hm, so here we have to answer the question of "should tests fail if OpenEye is _installed_, but not _licensed_?". 

I think they shouldn't. From a basic level, having an outdated OE license doesn't mean the OFF Toolkit is broken. And, the way we use it, the OE test configuration has a step to explicitly test the `oechem` import, so it will give a red status if the license is unavailable. 

So, for OETK, I think we should stick with our custom `is_available` method for deciding whether to skip the tests.

Reply:
I think I was asking a slightly different question ("should these tests run if it's installed, but in some way incorrectly?") but we have arrived at the same conclusion